### PR TITLE
Draw Chart with Apex

### PR DESCRIPTION
**Pull request by abharun**

### CODE REVIEW

File: src/components/ChartView.tsx [hunk 1]
```diff
@@ -1,102 +1,53 @@
...
+      text: stockInfo?.title,
     },
-    rightPriceScale: {
-      borderColor: "#485c7b",
+    subtitle: {
+      text: "Stock OHLC Prices",
     },
-    timeScale: {
-      borderColor: "#485c7b",
+    footnote: {
```
Comment (FreeFly0125) [inline src/components/ChartView.tsx:18]: is it default text for footnote?

File: src/components/SearchBar.tsx [hunk 2]
```diff
@@ -6,6 +6,8 @@ import { searchMock } from "../Mock/search";
 import { quoteMock } from "../Mock/quote";
 import { interdayMock } from "../Mock/interday";
 import { dailyMock } from "../Mock/daily";
+import { weeklyMock } from "../Mock/weekly";
```
Comment (FreeFly0125) [inline src/components/SearchBar.tsx:9]: Please create a new branch working with actual data.

**Review by FreeFly0125 — CHANGES_REQUESTED** [review]

File: src/components/ChartView.tsx (hunk 1 above)
Comment (abharun) [reply src/components/ChartView.tsx:18]: Will be fix.

File: src/components/SearchBar.tsx (hunk 2 above)
Comment (abharun) [reply src/components/SearchBar.tsx:9]: Sure.

File: src/components/ChartView.tsx (hunk 1 above)
Comment (abharun) [reply src/components/ChartView.tsx:18]: It's just for default.
When the stock info is received, it's replaced.
So don't care is okay.

**Review by FreeFly0125 — APPROVED** [review]

**Pull request merged**